{{cookiecutter.repostory_name}}/bin/restore-db-from-b2.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+
+if [ "$(basename "$0")" == 'bin' ]; then
+  cd ..
+fi
+
+if [ -z "$1" ]; then
+  echo "Pass backup file name/path (in B2) as the first argument"
+  exit 127
+fi
+
+if [[ -z "$BACKUP_B2_BUCKET" || -z "$BACKUP_B2_KEY_ID" || -z "$BACKUP_B2_KEY_SECRET" ]]; then
+  echo "Set BACKUP_B2_BUCKET, BACKUP_B2_KEY_ID and BACKUP_B2_KEY_SECRET env variables before running this script"
+  exit 127
+fi
+
+CONTAINER_NAME="{{ cookiecutter.repostory_name }}-backups-b2"
+docker build --quiet -t "$CONTAINER_NAME" tools/backup-b2
+
+docker run \
+  --mount type=bind,src="$(pwd)"/.backups,target=/root/.backups \
+  --env BACKUP_B2_BUCKET \
```

Comment:
Retrieving requires a different B2 key (default for backups will be write only key).
Also, requiring this to be set manually makes restore script a bit safer when run by mistake ;)

Reply:
@ppolewicz ?

Reply:
Correct with the different key, but BACKUP_B2_KEY_ID, BACKUP_B2_KEY_SECRET and FILE **ID** should all be mandatory command line arguments, not env variables. You should not use bucket name or file name, because B2 can store more than one file with the same name in the same bucket - file_id is a unique identifier of the file and that's what we should use.

It might mean we also need a script to list the existing backups: file names, file_ids, modTime, upload time, size, and an instruction for the operator to use it.

Reply:
Changed to take command line parameters, added separate script to list backups in B2.